chapters/en/slides/module2/module2_29.Rmd

```diff
@@ -153,21 +171,24 @@ Notes: Script here
 
 ---
 
-In the last plot, we used `.loc[:,'sugars']` to select a single column to the plot, however, we can show multiple mean column values in a single plot by selecting more columns. The columns `fat`, `fiber` and `protein` seem like good choices. 
-
-```{python out.width = '60%', fig.asp = .58}
-nutrition_plot = (df.groupby(by='mfr')
-                    .mean()
-                    .loc[:, ['fat', 'fiber', 'protein']]
-                    .plot.bar(title='Mean nutritrion value over different manufacturers')
-                 )
-nutrition_plot.set_ylabel('Content (in grams)', fontsize=9)
-nutrition_plot = nutrition_plot.set_xlabel('Manufacturer', fontsize=9)
-nutrition_plot
+The `x` and `y` labels from the bar plot on the previous slide are not very informative. Luckily, using the same syntax, we can set custom axis `titles` using the `title=""` argument. Let's see how this works by providing more informative titles to the plot from the previous slide.
```

Comment:
I wouldn't piggy back off the bins. Introduce this as a new concept. We must add title within the alt.X and alt.y ...

Reply:
Done